backend/src/main/java/gov/cdc/usds/simplereport/service/OrganizationInitializingService.java

```diff
@@ -66,6 +69,9 @@ public void initAll() {
     log.debug("Organization init called (again?)");
     Provider savedProvider = _providerRepo.save(_props.getProvider());
 
+    List<SupportedDisease> supportedDiseases =
```

Comment:
Mostly a curiosity for my sake, but is there a reason we're initializing the SupportedDiseaseRepository in the org init service? Put another way, is there a use for an organization to know what SupportDiseases are defined? Off the dome, there doesn't _seem_ to be any reason we need to know this at the org level, but lmk if I'm missing something

Reply:
The reason I added this was just to seed the test DB with some supported diseases - however, you're right that this `supportedDiseases` variable is unused, so I will change it to just save the diseases using the repo without storing in a variable!